backstop/tests/header.html

```diff
@@ -577,7 +574,6 @@ <h2>Slim</h2>
             <user-icon
               type="2"
               size="small"
-              showPlaceholder="true"
```

Comment:
I don't know about this change. I would have first tried to re-run tests to see if they'd pass

Reply:
I was trying to hide the image in the scenario JS, I tried:
`hideSelectors: ['user-icon']`
`hideSelectors: ['.iui-user-icon img']`
`hideSelectors: ['img']`
But wasn't able to hide the image from the tests. Am I doing it wrong?